### PR TITLE
FTP.WriteFile - Initial implementation

### PR DESCRIPTION
**Pull request by RikuVirtanen**
#22

### CODE REVIEW

File: Frends.FTP.WriteFile/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Frends.FTP.WriteFile
+Frends tasks for writing a file to the FTP(S) server.
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://opensource.org/licenses/MIT) 
+[![Build](https://github.com/FrendsPlatform/Frends.FTP/actions/workflows/WriteFile_build_and_test_on_main.yml/badge.svg)](https://github.com/FrendsPlatform/Frends.FTP/actions)
+![MyGet](https://img.shields.io/myget/frends-tasks/v/Frends.FTP.WriteFile)
```
Comment (ttossavainen) [inline Frends.FTP.WriteFile/README.md]: This row can be deleted.

File: Frends.FTP.WriteFile/Frends.FTP.WriteFile/Definitions/Input.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    /// <summary>
+    /// Text content to be written.
+    /// </summary>
+    /// <example>This is test file</example>
+    [DisplayFormat(DataFormatString = "Text")]
+    public string Content { get; set; }
+
+    /// <summary>
+    /// Enables Task to create destination directories if they don't exist.
+    /// </summary>
```
Comment (ttossavainen) [inline Frends.FTP.WriteFile/Frends.FTP.WriteFile/Definitions/Input.cs:29]: Please add an example.

File: Frends.FTP.WriteFile/Frends.FTP.WriteFile/Definitions/Connection.cs [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+    /// <summary>
+    /// Use implicit encryption.
+    /// </summary>
+    Implicit,
+
+    /// <summary>
+    /// Tries to use FTPS encryption and falls back to plain text FTP.
+    /// </summary>
+    Auto,
+}
```
Comment (ttossavainen) [inline Frends.FTP.WriteFile/Frends.FTP.WriteFile/Definitions/Connection.cs]: There appears to be an "Enums" folder. Would it be possible to move these enums to that folder, or alternatively, move all into a single file?

File: Frends.FTP.WriteFile/Frends.FTP.WriteFile/WriteFile.cs [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+{
+    /// <summary>
+    /// Writes file to the FTP(S) server.
+    /// [Documentation](https://tasks.frends.com/tasks/frends-tasks/Frends.FTP.WriteFile)
+    /// </summary>
+    /// <param name="connection">Connection parameters</param>
+    /// <param name="input">Input parameters</param>
+    /// <param name="cancellationToken">Token to stop task. This is generated by Frends.</param>
+    /// <returns>Object { string Path, double SizeBytes }</returns>
+    public static Result WriteFile([PropertyTab] Connection connection, [PropertyTab] Input input, CancellationToken cancellationToken)
```
Comment (ttossavainen) [inline Frends.FTP.WriteFile/Frends.FTP.WriteFile/WriteFile.cs:27]: Please remove unused parameter cancellationToken.

Comment (RikuVirtanen) [reply Frends.FTP.WriteFile/Frends.FTP.WriteFile/WriteFile.cs:27]: Added usage for the cancellationToken.

**Review by ttossavainen — APPROVED** [review]

**Pull request merged**